Assignment 5.Rmd

```diff
@@ -65,16 +67,22 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 
 ```{r}
 #Upload new data
-
-D2 <- 
+library(dplyr)
+D2 <- read.csv("intelligent_tutor_new.csv")
 
 #Generate predicted advice using the predict() command for new students based on tree generated from old students
 
-D2$prediction <- 
+D2$prediction <- predict(score_ctree,D2)
 
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+```{r}
+D2$advice <- ifelse(D2$score>=0.8,1,ifelse(D2$score<=0.5,3,2))
+accuracy<-nrow(filter(D2,prediction==1))/nrow(D2)
+
+#The predicted score is 65% accurate. 
```

Comment:
Elaborate on it, as in, specify whether the model overfits or whether it is generalizable. Also, calculate the error rate of the model predictions.